secure_message/services/party_service.py

```diff
@@ -59,7 +60,7 @@ def get_user_details(self, uuid):
             return party_dict, 200
 
         if uuid not in self.__users_cache:
-            logger.info("Executing call to the  party service ")
+            logger.info("Party Service: retrieve party data using", uuid=uuid)
```

Comment:
We can drop the "using" here as the `uuid` is now separate from this statement.

Reply:
nope sorry I am not going to change it.